actions/vmwarelib/actions.py

```diff
@@ -79,6 +84,39 @@ def _connect(self, vsphere):
         atexit.register(connect.Disconnect, si)
         return si
 
+    def _connect_rest(self, vsphere):
+        connection = self._get_connection_info(vsphere)
+
+        try:
+            session = requests.Session()
+            session.verify = False
+            session.auth = (connection['user'], connection['passwd'])
+
+            login_url = "https://%s/rest/com/vmware/cis/session" % connection['host']
+
+            session.post(login_url)
+
+        except Exception as e:
```

Comment:
re-raising here is pointless... simply remove the try/catch block if this is all you're going to do